data.sql

```diff
@@ -11,3 +11,31 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
  
  INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
  VALUES( 4, 'Devimon', '2017-05-12', 5, TRUE, 11.00);
+
+
+ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+ VALUES(5, 'Charmander', '2020-02-08', 0, FALSE, 11.00);
+
+
+ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+ VALUES(6, 'Plantmon', '2021-11-15', 2, TRUE, 5.70);  
+
+ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+ VALUES(7, 'Squirtle', '1993-04-02', 3, FALSE, 12.13);
+
+
+  INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+  VALUES(8, 'Angemon', '2005-06-12', 1, TRUE, 45.00);
```

Comment:
- Please, ensure you've used the correct given values as these are missing the negative sign denoting they're floating numbers to meet the project requirement.
`-11.0`, ` -5.7`, `-12.13` and `-45.0`